src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/BdioUploader.groovy

```diff
@@ -59,6 +62,11 @@ class BdioUploader {
 
         String hubDetectVersion = detectInfo.detectVersion
         PhoneHomeRequestBody phoneHomeRequestBody = phoneHomeDataService.createInitialPhoneHomeRequestBodyBuilder(ThirdPartyName.DETECT, hubDetectVersion, hubDetectVersion).build()
+
+        Set<BomToolType> applicableBomTools = detectProject.getApplicableBomTools();
+        String applicableBomToolsString = StringUtils.join(applicableBomTools, ", ");
+        phoneHomeRequestBody.getInfoMap().put("bomToolTypes", applicableBomToolsString);
```

Comment:
Ideally you should create the body builder and modify that, then build the body rather than building and modifying the generated body.